### PR TITLE
Add simple face numbering in 2D

### PR DESCRIPTION
**Pull request by jandrej**
This PR adds a very simple face numbering in 2D for debugging purposes

### CODE REVIEW

File: lib/vssolution.cpp [hunk 1]
```diff
@@ -1815,6 +1815,50 @@ void VisualizationSceneSolution::PrepareVertexNumbering2()
    updated_bufs.emplace_back(&v_nums_buf);
 }
 
+void VisualizationSceneSolution::PrepareFaceNumbering()
+{
+   f_nums_buf.clear();
+
+   if (mesh->Dimension() != 2)
+   {
+      return;
```
Comment (tzanio) [inline lib/vssolution.cpp:1824]: This check is unnecessary, everything in this file is already 2D.

File: lib/vssolution.hpp [hunk 2]
```diff
@@ -154,7 +156,7 @@ class VisualizationSceneSolution : public VisualizationSceneScalarData
    void ToggleDrawOrdering() { draworder = (draworder+1)%5; }
 
    // 0 - none, 1 - elements, 2 - vertices
```
Comment (tzanio) [inline lib/vssolution.hpp:158]: update comment

File: lib/vssolution.cpp [hunk 3]
```diff
@@ -1815,6 +1815,50 @@ void VisualizationSceneSolution::PrepareVertexNumbering2()
    updated_bufs.emplace_back(&v_nums_buf);
 }
 
+void VisualizationSceneSolution::PrepareFaceNumbering()
```
Comment (tzanio) [inline lib/vssolution.cpp:1818]: Should it be `face` or `edge` numbering?

File: lib/vssolution.cpp [hunk 4]
```diff
@@ -1908,6 +1952,7 @@ void VisualizationSceneSolution::PrepareNumbering()
 {
    PrepareElementNumbering();
    PrepareVertexNumbering();
+   PrepareFaceNumbering();
```
Comment (tzanio) [inline lib/vssolution.cpp:1955]: This seems out-of-order for me, pick either `vertex-edge/face-element` or `element-edge/face-vertex`

**Review by tzanio — APPROVED** [review]
Minor suggestions

**Comment by tzanio** [comment]
Please mention also in `CHANGELOG`

**Comment by tzanio** [comment]
ping: @jandrej

File: CHANGELOG [hunk 5]
```diff
@@ -21,6 +21,8 @@ Version 4.1.1 (development)
 
 - In 2D, save and restore solution's value range when using keys 'e' and 'b'.
 
+- A face numbering option is now available in 2D.
```
Comment (tzanio) [inline CHANGELOG:24]:
```suggestion
- An edge numbering option is now available in 2D.
```

**Review by tzanio — APPROVED** [review]
Thanks @jandrej !

**Comment by rw-anderson** [comment]
LGTM

**Review by rw-anderson — APPROVED** [review]

**Pull request merged**